### ISSUE TITLE
Dice Success calculation problem

### ISSUE THREAD

**Issue by CepheiA2** [mordachai/household-csb#5]
Hello,
There is something  i don't undestand about the dice macro calculation:

check the attachment first.

As you can see in the screenshot.
On the first roll i got a basic and a critical  success. 

then  i rerolled the three remaining dice, 

i should have got 2 basic and 1 critical, according to the chat image, but the success text says that i got 1x basic and 2x critical instead. I find out this is a recurrent behavior.

There is something i am missing or it's a bug?
![screenshot](https://github.com/mordachai/household-csb/assets/85868381/d66bb23e-565c-4f5e-ab5f-d6515f7cf690)

**Comment by kemain** [comment]
yup, i noticed that the bug recurs when you reroll the dice and you obtain a pair of dice of the same seed(number),as one of the one you got in the remaining dice.
I attach a screenshot to prove my theory
![immagine](https://github.com/mordachai/household-csb/assets/23489737/e76ddb58-4570-4596-a94a-b85ad1e548d6)
